geometry_msgs/Pose pose
---
bool status